openfisca_core/simulations.py

```diff
@@ -132,12 +132,13 @@ def clone(self, debug = False, debug_all = False, trace = False):
             if key not in ('debug', 'debug_all', 'trace', 'entities'):
```

Comment:
Not sure, but I think we don't want the `debug`, `debug_all` and `trace` attributes of the original simulation to be copied to the new one. Instead, we only take their value from the parameters of the `clone` method.